x-pack/heartbeat/monitors/browser/synthexec/execmultiplexer_test.go

```diff
@@ -45,11 +45,20 @@ func TestExecMultiplexer(t *testing.T) {
 			})
 		}
 
-		testEvents = append(testEvents, &SynthEvent{
-			Journey:              journey,
-			Type:                 "journey/end",
-			TimestampEpochMicros: time,
-		})
+		// We want one of the test journeys to end with a cmd/status indicating it failed
+		if jIdx != 4 {
+			testEvents = append(testEvents, &SynthEvent{
```

Comment:
As per previous comment, lets add both and see what happens with summary creation.